Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -91,11 +91,27 @@ struct SamplePaywallsList: View {
             #if CUSTOMER_CENTER_ENABLED
             CustomerCenterView()
             #endif
+        #if PAYWALL_COMPONENTS
+        case .componentPaywall(let data):
+            ComponentsView(locale: .current, components: data.components)
+        #endif
```

Comment:
interesting that we didn't end up with the same structure for the `#if`s for customer center and paywall components for this

Reply:
The issue is that the associated value for the paywall components case is very much a `PAYWALL_COMPONENTS` entity.